src/components/Post/Post.tsx

```diff
@@ -1,10 +1,10 @@
 import "./Post.css";
 import {Avatar, Paper} from '@mui/material';
-import {deepPurple} from '@mui/material/colors';
+import { deepPurple} from '@mui/material/colors';
 // import ThumbUpIcon from '@mui/icons-material/ThumbUp';
```

Comment:
remove unnecessary icons

Reply:
removed

Reply:
not sure is it useful for further enhancement